org.sparkleshare.SparkleShare.json

```diff
@@ -0,0 +1,130 @@
+{
+    "app-id": "org.sparkleshare.SparkleShare",
+    "sdk": "org.gnome.Sdk",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "command": "sparkleshare",
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=home",
+        "--share=ipc",
+        "--share=network",
+        "--socket=wayland",
+        "--socket=x11",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*"
+    ],
+    "modules": [
+        {
+            "name": "mono",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.mono-project.com/sources/mono/mono-4.8.1.0.tar.bz2",
+                    "sha256": "18cb38a670e51609c36c687ed90ad42cfedabeffd0a2dc5f7f0c46249eb8dbef"
+                }
+            ]
+        },
+        {
+            "name": "openssh",
+            "config-opts": ["--without-pie"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://mirror.exonetric.net/pub/OpenBSD/OpenSSH/portable/openssh-7.5p1.tar.gz",
+                    "sha256": "9846e3c5fab9f0547400b4d2c017992f914222b3fd1f8eee6c7dc6bc5e59f9f0"
+                }
+            ]
+        },
+        {
+            "name": "git",
+            "make-args": ["NO_PERL=1"],
+            "make-install-args": ["NO_PERL=1"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.kernel.org/pub/software/scm/git/git-2.14.1.tar.xz",
+                    "sha256": "6f724c6d0e9e13114ab35db6f67e1b2c1934b641e89366e6a0e37618231f2cc6"
+                }
+            ]
+        },
+        {
+            "name": "git-lfs",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/git-lfs/git-lfs/releases/download/v2.3.0/git-lfs-linux-amd64-2.3.0.tar.gz",
+                    "sha256": "5913ed4d023efe30a3f8f536db7cd97c7b76ba68b189baef8428b1d71d82f2f4"
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir --parents /app/libexec/git-core/",
+                "cp git-lfs /app/libexec/git-core/"
+            ]
+        },
+        {
+            "name": "gtk-sharp3",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mono/gtk-sharp.git",
+                    "commit": "9ef7d3f394b9d485674c59e92db24ac66fd9f16b"
+                }
+            ]
+        },
+        {
+            "name": "dbus-sharp",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/hbons/dbus-sharp",
```

Comment:
Are you planning on getting your changes upstreamed?

Reply:
Using the upstream for dbus-sharp now. for dbus-sharp-glib there are still changes needed for the build that need to be merged upstream. I've created a PR, but the binding projects are pretty much dead. A GNOME hackfest is being organised to fix this.